web/next.config.js

```diff
@@ -2,6 +2,7 @@
 const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
-}
+  GRAPHQL_URL: "https://api-mumbai.lens.dev/",
```

Comment:
https://gitlab.com/jtco-io/pub/pg-gql-next/-/blob/main/apps/web/next.config.js#L16

Reply:
This needs to be under publicRuntime